### ISSUE TITLE
Universal Intro Text: Update Contrast of text - 1.4.3 Contrast (Minimum) 

### ISSUE THREAD

**Issue by syarwick** [stephenravenscraft/loyola-template-4.0#61]
The body paragraph link on the maroon intro text is also maroon, which means the link is invisible and cannot be seen by users. 

ADDITIONALLY, the table headers appear as black text on a maroon background, which does not have a sufficient contrast of 4.5:1.
